test/unit/services/test-reduction-service.js

```diff
@@ -71,15 +106,20 @@ beforeEach(function () {
 describe('services/reductions-service', function () {
   describe('Get reductions', function () {
     it('should create the result object with the right information', function () {
-      reductionService.getReductions(1, orgUnitConstant.OFFENDER_MANAGER.name)
+      getReductions.resolves([])
+      return reductionService.getReductions(1, orgUnitConstant.OFFENDER_MANAGER.name)
         .then(function (result) {
+          assert(getReductions.called)
+          assert(getBreadcrumbsStub.called)
           expect(result.title).to.equal('John Doe')
           expect(result.subTitle).to.equal('Offender Manager')
-          assert(getBreadcrumbsStub.called)
+          expect(result.activeReductions).to.eql(activeReductions)
+          expect(result.scheduledReductions).to.eql([])
+          expect(result.archivedReductions).to.eql(archivedReductions)
```

Comment:
How are deleted reductions handled?  Are they retireved from database? If so it might be good to test around that.

Reply:
Good point. The view shouldn't have to deal with them, but the helper might, @ddebella

Reply:
Deletes are not removed from the table, but are marked with DELETE status. These are checked  by the helper to ensure any DELETE status are not returned.